api/src/main/java/org/eclipse/microprofile/health/package-info.java

```diff
@@ -21,6 +21,6 @@
 /**
  * <p>Microprofile Health</p>
  **/
-@org.osgi.annotation.versioning.Version("1.0")
+@org.osgi.annotation.versioning.Version("1.1")
```

Comment:
I am not much experienced with OSGi but should the version be aligned with the release? So 2.0?

Reply:
I already asked @Emily-Jiang, apparently this doesn't have to follow spec version

Reply:
@antoinesd was right. The package version just maintain the status of api in that package. If nothing is changed in that package, the package version will stay the same even thought  it is a major release.